tests/test_aliases.py

```diff
@@ -405,10 +405,16 @@ class Model(BaseModel):
 def test_aliases_json_schema():
     class Model(BaseModel):
         x: str = Field(alias='x_alias', validation_alias='x_val_alias', serialization_alias='x_ser_alias')
+        y: str = Field(validation_alias=AliasChoices('y_alias', 'another_alias'))
+        z: str = Field(validation_alias=AliasChoices(AliasPath('z_alias', 'even_another_alias'), 'and_another'))
```

Comment:
I's rather separated cases using `pytest.mark.parametrize`. This allows easier debugging and shows which case is failing exactly.